experiment_without_Keithley.py

```diff
@@ -0,0 +1,158 @@
+
+        data.set_value('Time', t)
+        data.set_value('Temperature', T)
+        data.set_value('Field', F)
+        data.set_value('Chamber Status', C)
+        data.set_value('Resistance', res)
+        data.write_data()
+
+        end_time = time.time()
+        time_diff = end_time - start_time
+
+        if time_diff > wait:
+            raise Exception('The inner sweep takes too long!')
+
+        # poll data at roughly equal intervals based on points/ramp
+        time.sleep(wait - time_diff)
+
+with mpv.Client() as client:
+    client.open()
+    
+    print('client accepted')
+    
+    # Set 5 K and 0 Oe
+    print('Setting 5 K and -50,000 Oe')
+    client.set_temperature(
+        5,
+        3000, #set back to 3
+        client.temperature.approach_mode.fast_settle
+    )
+    client.set_field(
+        -50000.0,
+        20,
+        client.field.approach_mode.linear,
+        client.field.driven_mode.driven
+    )
+    
+    # Wait for 60 seconds after temperature and field are stable
+    print('Waiting for Temp and Field')
+    client.wait_for(
+        60,
+        timeout_sec=0,
+        bitmask=client.temperature.waitfor | client.field.waitfor
+    )
+
+        # Wait for dewar pressure
+    time.sleep(60 * 60)
+    
+    # Configure resistivity measurement
+    client.resistivity.bridge_setup(
+        bridge_number=1,
+        channel_on=True,
+        current_limit_uA=50,
+        power_limit_uW=500,
+        voltage_limit_mV=1000
+    )
+    time.sleep(5)  # recommended time to sleep for config above to take place
+    
+    # set the current across the hallbar TODO: ref paper Sammak et. al.
+    client.resistivity.set_current(bridge_number=1, current_uA=50)
+    
+    # configure the MultiVu columns
+    data = mpv.DataFile()
+    data.add_multiple_columns(['Time', 'Temperature', 'Field', 'Chamber Status', 'Resistance'])
+    data.create_file_and_write_header('Resistance.dat', 'Hallbar data')
+    
+    # Polling temperature/field and performing resistivity measurement 
+    # during a field ramp from -50,000 to 0.0 Oe at 20 Oe/sec
+    Scan_Field(0)  
+    
+    print("waiting for dewar pressure (60 mins)")
+    time.sleep(60 * 60)  # wait for dewar pressure to stabilize
+    
+    # Polling temperature/field and performing resistivity measurement 
+    # During a field ramp from 0 to 50,000 Oe at 20 Oe/sec
+    Scan_Field(50000)
+    
+    print("waiting for dewar pressure (60 mins)")
+    time.sleep(60 * 60)  # wait for dewar pressure to stabilize
+    
+    # Polling temperature/field and performing resistivity measurement 
+    # During a field ramp from 50,000 to 0 Oe at 20 Oe/sec
+    Scan_Field(0)
+    
+    print("waiting for dewar pressure (60 mins)")
+    time.sleep(60 * 60)  # wait for dewar pressure to stabilize
+    
+    # Polling temperature/field and performing resistivity measurement 
+    # During a field ramp from 0 to -50,000 Oe at 20 Oe/sec
+    Scan_Field(-50000)
+    
```

Comment:
Should we wait for the dewar pressure here as well?

Reply:
```suggestion
    # Polling temperature/field and performing resistivity measurement 
    # During a field ramp from 0 to -50,000 Oe at 20 Oe/sec
    Scan_Field(-50000)
    
    time.sleep(60*60)
```